magprime/templates/magprime_dept_checklist/allotments.html

```diff
@@ -0,0 +1,191 @@
+{% extends "base-admin.html" %}
```

Comment:
This file is also taken from ``magclassic`` with a number of changes that I went over with Craig.